flash-messages/env.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL="sqlite:flash-messages-dev.db"
```

Comment:
I'd prefer if standard operating procedure was to make it so .env.example is copy-pastable for development environments; except when a user needs to generate a key for an outside service.